entrypoint.sh

```diff
@@ -55,6 +55,7 @@ if [ ! -f "${DOCKER_CACHE_DIR}/.install_complete" ]; then
             -DIREE_HAL_DRIVER_DEFAULTS=OFF \
             -DIREE_HAL_DRIVER_LOCAL_SYNC=ON \
             -DIREE_HAL_DRIVER_LOCAL_TASK=ON \
+            -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
```

Comment:
It's nice to be able to take a look at the runtime code in VSCode.

Reply:
I wonder how I was able to use clangd to look at runtime code without this so far. I also see a `~/code/.cache/docker/iree/build/compile_commands.json` generated by default (without this flag).

Reply:
Yep, it looks like this is [not necessary](https://github.com/iree-org/iree/blob/4284be655aa51c3f0d690f8d616a7cd431ef3e99/CMakeLists.txt#L17). I remember my intellesense not working and I thought it was this that fixed it, but that must have been a red herring. I'll remove this.